package.json

```diff
@@ -9,7 +9,9 @@
     "lint-fix": "npm run eslint -- --fix && npm run prettier -- --write",
     "eslint": "eslint './**/*.js'",
     "prettier": "prettier './**/*.js' './**/*.sol' './**/*.md'",
-    "test": "lerna run --stream --concurrency=1 test",
+    "test": "yarn test-no-concurrent && yarn test-concurrent",
```

Comment:
Do you think it would make sense to have the default "test" run everything serially and have a concurrent option that runs some of the modules concurrently?

Not strongly opinionated, it could just be nice to not have our default test command require substantial system resources. The parallel logs can also be somewhat confusing.

Thoughts?

Reply:
ye, this is a good point. the parallel logs are basically impossible to read. I'll change this to have a separate entry point that CI uses.